package.json

```diff
@@ -12,7 +12,6 @@
     "typecheck": "turbo typecheck --filter=./devtools/* --filter=./packages/*",
     "test": "turbo test --filter=./devtools/* --filter=./packages/*",
     "test:watch": "turbo test:watch --filter=./devtools/* --filter=./packages/*",
-    "test:e2e": "pnpm --filter='fixture-*' run test:e2e",
```

Comment:
Please restore this script. I'm using it to test e2e locally on MacOS and it is working fine.

Reply:
I see

Reply:
Restored